scss/style.scss

```diff
@@ -1,14 +1,8 @@
 // ==========================================================================
 // CSS Chassis
 // ==========================================================================
+// This just add normalize to the build see lint.scss for expaination.
 
 @import
-	"_utilities/clearfix",
-	"_utilities/colors";
-
-@import
-	"atoms/icons/icons",
-	"atoms/typography/typography";
-
-@import
-	"views/main";
+	"external/normalize.css/normalize.scss",
+	"lint.scss";
```

Comment:
Also as calling it `lint.scss` will mean we're going to have a compiled lint.css file as well as a style.css.  If you're going to pull all of that out of `styles.scss`, then probably call it  `_lint.scss` so it's not compiled in two locations.